src/packet/rustq/src/lib.rs

```diff
@@ -195,7 +195,8 @@ impl WrapperPacketQueue {
     }
 }
 
-use hwfq::{scheduler::htb::ClassedTokenBucket, scheduler::Drr, Pkt, Scheduler};
+use hwfq::{scheduler::htb::ClassedTokenBucket, scheduler::drr::Drr, Pkt, Scheduler};
```

Comment:
this change shouldn't be necessary: https://github.com/akshayknarayan/hwfq/blob/master/src/scheduler/mod.rs#L47

Reply:
I see, you made DRR not a `pub use` at https://github.com/akshayknarayan/hwfq/pull/4/files#diff-51d143b69b6ca8b0c0417b328de29bbb8ad98f691bc78019803808b22578e016R47.